### PR TITLE
Release notes for 111

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered RelNotes page for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2480 [[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/ReleaseNotes111/microsoft-edge/webview2/release-notes.md)]

AB#43632898

### CODE REVIEW

**Review by novac42 — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -36,13 +37,524 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+
+Supplemental `TextureStream*` interfaces:
+* CoreWebView2TextureStreamStartRequestedEventHandler<!-- todo: is there such? not listed in version delta list.  found in Win32 -->
+* CoreWebView2TextureStreamStoppedEventHandler<!-- todo: is there such? not listed in version delta list.  found in Win32 -->
+* CoreWebView2TextureStreamErrorReceivedEventHandler<!-- todo: is there such? not listed in version delta list.  found in Win32 -->
+
+ErrorReceivedEventArgs:
+* [CoreWebView2TextureStreamErrorReceivedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorreceivedeventargs?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2TextureStreamErrorReceivedEventArgs.Kind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorreceivedeventargs.kind?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2TextureStreamErrorReceivedEventArgs.texture Property](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorreceivedeventargs.texture?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)<!-- todo: uppercase "Texture"? -->
```
[the remark this answers is not in the document]
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md:371]: Should keep lowercase `texture` for now. We do plan to fix the API name to `Texture` in the future release.

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -36,13 +37,526 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+    * [CoreWebView2TextureStream.StartRequested Event](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestream.startrequested?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+    * [CoreWebView2TextureStream.Stop Method](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestream.stop?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+    * [CoreWebView2TextureStream.Stopped Event](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestream.stopped?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+    * [CoreWebView2TextureStream.WebTextureReceived Event](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestream.webtexturereceived?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+    * [CoreWebView2TextureStream.WebTextureStreamStopped Event](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestream.webtexturestreamstopped?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+Supplemental `TextureStream*` interfaces:
+* CoreWebView2TextureStreamStartRequestedEventHandler<!-- todo: is there such? not listed in version delta list.  found in Win32 -->
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:364]: No EventHandler or CompletedHandler in .NET or WinRT.

General event pattern

- Win32: `add/remove_XYZ` + `XYZEventHandler`
- .NET/WinRT: `XYZ` event

General async method pattern

- Win32: `XYZ` method + `XYZCompletedHandler`
- .NET/WinRT: `XYZAsync`

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+    * [CoreWebView2TextureStream.WebTextureReceived Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texturestream?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#webtexturereceived)
+    * [CoreWebView2TextureStream.WebTextureStreamStopped Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texturestream?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#webtexturestreamstopped)
+
+ErrorReceivedEventArgs:
+* [CoreWebView2TextureStreamErrorReceivedEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texturestreamerrorreceivedeventargs?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2TextureStreamErrorReceivedEventArgs.Kind Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texturestreamerrorreceivedeventargs?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#kind)
+   <!-- * [CoreWebView2TextureStreamErrorReceivedEventArgs.texture Property]() todo: not found -->
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:434]: Not available in WinRT.

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+   * [CoreWebView2TextureStreamErrorKind.CoreWebView2TextureStreamErrorTextureInUse Enum Value](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texturestreamerrorkind?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true)
+
+The `Texture` interface that the host writes to so that the Renderer will render on it:
+* [CoreWebView2Texture Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texture?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2Texture.Resource Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texture?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#resource)
+   * [CoreWebView2Texture.Timestamp Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texture?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#timestamp)
+   <!-- * [CoreWebView2Texture.Handle Property]()  todo: not found, moved to bottom of list -->
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:451]: Not available in WinRT.

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+
+The received `WebTexture` interface that the Renderer writes to so that the host will read on it:
+* [CoreWebView2WebTexture Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2webtexture?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2WebTexture.Resource Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2webtexture?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#resource)
+   * [CoreWebView2WebTexture.Timestamp Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2webtexture?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#timestamp)
+   <!-- * [CoreWebView2WebTexture.Handle Property]()  todo: not found, moved to bottom of list -->
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:457]: Not available in WinRT.

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+   * [ICoreWebView2ExperimentalTextureStreamWebTextureReceivedEventArgs::get_WebTexture](/microsoft-edge/webview2/reference/win32/icorewebview2experimentaltexturestreamwebtexturereceivedeventargs?view=webview2-1.0.1724-prerelease&preserve-view=true#get_webtexture)
+* [ICoreWebView2ExperimentalTextureStreamWebTextureStreamStoppedEventHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentaltexturestreamwebtexturestreamstoppedeventhandler?view=webview2-1.0.1724-prerelease&preserve-view=true)
+
+<!-- todo: add ERROR_KIND enum equiv of .net TextureStream error kind enum -->
+
+Other interfaces (`RenderAdapter`):
+* [ICoreWebView2ExperimentalRenderAdapterLUIDChangedEventHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalrenderadapterluidchangedeventhandler?view=webview2-1.0.1724-prerelease&preserve-view=true)<!-- todo: does this belong in this "ICoreWebView2ExperimentalTextureStream*" group? -->
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:507]: This is related to texture stream API and belongs here.

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+Added support for cookie manager:
+* [CoreWebView2Profile.CookieManager Property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.cookiemanager?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+Add support for managing profile deletion:
+* [CoreWebView2Profile.Delete Method](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.delete?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+* [CoreWebView2Profile.Deleted Event](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.deleted?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2 Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true)
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:546]: The 2 lines are included by mistake - pls remove CoreWebView2 Class and CoreWebView2.Profile Property.

Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md:546]: Same for .NET section too.

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+* [CoreWebView2TextureStreamErrorReceivedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorreceivedeventargs?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2TextureStreamErrorReceivedEventArgs.Kind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorreceivedeventargs.kind?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2TextureStreamErrorReceivedEventArgs.texture Property](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorreceivedeventargs.texture?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+WebTextureReceivedEventArgs:
+* [CoreWebView2TextureStreamWebTextureReceivedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamwebtexturereceivedeventargs?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+* [CoreWebView2TextureStreamWebTextureReceivedEventArgs.WebTexture Property](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamwebtexturereceivedeventargs.webtexture?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+<!-- todo: not added yet in Win32: -->
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:388]: https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2experimental-idl?view=webview2-1.0.1724-prerelease#corewebview2_texture_stream_error_kind

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -36,13 +37,587 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+
+* [CoreWebView2PermissionKind.WindowManagement Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2PermissionKind.WindowManagement Enum Value](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2permissionkind?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true)
+
+##### [Win32/C++](#tab/win32cpp)
+
+<!-- todo: add Win32 link?
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:297]: https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.1724-prerelease#corewebview2_permission_kind

File: microsoft-edge/webview2/release-notes.md (hunk 6 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: removed todo

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -36,13 +37,545 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+    * [CoreWebView2TrackingPreventionLevel.Strict Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2trackingpreventionlevel?view=webview2-dotnet-1.0.1661.34&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1661.34&preserve-view=true)
+   * [CoreWebView2EnvironmentOptions.EnableTrackingPrevention Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1661.34&preserve-view=true#enabletrackingprevention)
+* [CoreWebView2Profile Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1661.34&preserve-view=true)
+   * [CoreWebView2Profile.PreferredTrackingPreventionLevel Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1661.34&preserve-view=true#preferredtrackingpreventionlevel)
+
```
[the remark this answers is not in the document]
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md:197]: 1. Yes.
2. Yes. https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.1661.34#corewebview2_tracking_prevention_level

File: microsoft-edge/webview2/release-notes.md [hunk 11]
```diff
@@ -36,13 +37,545 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.GetDeferral Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#getdeferral)
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.InitiatingOrigin Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#initiatingorigin)
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.IsUserInitiated Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#isuserinitiated)
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.Uri Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#uri)
+
```
Comment (dan-wesley) [inline microsoft-edge/webview2/release-notes.md:303]: blank line - inconsistent with other lists

File: microsoft-edge/webview2/release-notes.md [hunk 12]
```diff
@@ -36,13 +37,545 @@ Make sure to re-compile your WebView2 app after updating the WebView2 SDK NuGet
...
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.InitiatingOrigin Property](/dotnet/api/microsoft.web.webview2.core.corewebview2launchingexternalurischemeeventargs.initiatingorigin?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.IsUserInitiated Property](/dotnet/api/microsoft.web.webview2.core.corewebview2launchingexternalurischemeeventargs.isuserinitiated?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.Uri Property](/dotnet/api/microsoft.web.webview2.core.corewebview2launchingexternalurischemeeventargs.uri?view=webview2-dotnet-1.0.1724-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.LaunchingExternalUriScheme Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1724-prerelease&preserve-view=true#launchingexternalurischeme)
```
Comment (dan-wesley) [inline microsoft-edge/webview2/release-notes.md:296]: rendered doc shows blank line between Event and Class

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:352]: i reformatted all, adding space between types

gh rendered
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/ReleaseNotes111/microsoft-edge/webview2/release-notes.md

**Review by peiche-jessica — APPROVED** [review]

**Comment by dan-wesley** [comment]
All the links work

**Review by dan-wesley — COMMENTED** [review]
links tested and ok

**Review by dan-wesley — APPROVED** [review]

**Pull request merged**